script/app/settings-test.py

```diff
@@ -0,0 +1,123 @@
+"""
```

Comment:
Instead of duplicating a lot of the settings like:
```
- app/settings-dev.py
- app/settings-test.py
```

Change the `settings.py` into `base.py` and then create a test settings file that imports the base:
```
- app/
 -- settings/
  --- base.py
  --- test.py
```

So `test.py` would look like:
```
from script.settings.base import *

...
# then whatever override of the settings of base that are required for test
```

Reply:
Thanks for your suggestions!